src/app/_components/admincomponents/transport-request-modal.module.scss

```
@@ -0,0 +1,37 @@
+  font-weight: $headings-font-weight;
+}
+
+.fieldHeadings {
+  color: $color-cherry-red;
+  font-weight: $headings-font-weight;
+  font-size: large;
+  padding-bottom: $padding-bottom-amount;
+}
+
+.pastHistoryHeading {
+  @extend .fieldHeadings;
```

Stylelint (17.3.0)

[error] Expected a placeholder selector (e.g. %placeholder) to be used in `@extend` (scss/at-extend-no-missing-placeholder)

(scss/at-extend-no-missing-placeholder)